### DISCUSSION TITLE
FILE CANNOT BE SAVED BECAUSE THE FILE SIZE EXCEEDS THE PERMITTED LIMIT

### DISCUSSION THREAD

**Discussion by realperest** [utmapp/UTM#5188] [UTM Help (macOS)]
Hello.
I want to share a flash disk, open it from UTM and copy a file from there to UTM WIN side.

About 10 gb. outlook.pst file

When I copy paste, I get an error message that the FILE CANNOT BE SAVED BECAUSE THE FILE SIZE EXCEEDS THE PERMITTED LIMIT.

Is there a solution for this?

**Comment by conath** [comment]
How are you copying the file? Via the folder sharing feature, or via USB? If the file is on a USB flash drive, you could try using the USB sharing feature instead of folder share, as that doesn't have a file size limitation. (the limit here is probably the Windows WebDAV client IIRC)

**Comment by mtissington** [comment]
I'm seeing this issue to - using the folder sharing feature.

**Comment by Qhilm** [comment]
I see the issue too. I have mapped a macOS folder using Spice webdav. I can successfully browse it. I am doing drag & drop from the network drive Z: (SPICE webdav mapped) to desktop and getting this error.

![image](https://github.com/utmapp/UTM/assets/3350433/98e04555-225d-4d19-8887-b0bc27a148ca)


File size: the limit seems to be 50MB (42MB was fine, 52MB throws the error)
host: macOS 13.5
guest: Windows 11 ARM
UTM 4.3.5 (87)

**Comment by Qhilm** [comment]
Solved: I increased webdav size limit in windows registry (FileSizeLimitInBytes) and it works now.

See here for step by step:
https://www.imageway.com/2017/email-hosting-blog/email-hosting-tutorials/increase-webdav-file-limit-in-windows

**Reply by LukeMurphy** [reply]
Thanks ++

**Reply by orvvro** [reply]
Awesome, thanks a lot

**Comment by The-Johnson** [comment]
Ok, so any idea how we solve this for a 5GB file? You can only increase the WebDAV limit so much :(

**Comment by mustafaabobakr** [comment]
## For future reference 
> [!TIP]
> To solve this `The file size exceeds the limit allowed and cannot be saved`
> ![image](https://github.com/user-attachments/assets/f5d0605e-4cd7-4017-8a90-da2a8642a035)

1. Open regedit by 
   - Press `command + R` 
   - Type `regedit`
3. in the address bard go copy the following and press enter
```
Computer\HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\WebClient\Parameters
```
4. Open the `FileSizeLimitinBytes`
5. Add max value `ffffffff`

![image](https://github.com/user-attachments/assets/2105f691-78fd-4946-9abc-665d94813ea2)

**Reply by xX-Mordran-Xx** [reply]
Worked just fantastic for me!  Thank you soooo much!
UTM macOS v4.6.4 ARM64 using a Windows 11 24H4 ARM64 VM

**Reply by m3di** [reply]
and then restart after +1

**Reply by resnyanskiy** [reply]
This solution should be added to the https://docs.getutm.app/guest-support/windows/#spice-webdav
How it can be done?

**Comment by flusher35** [comment]
doesn't work for me. set to ffffffff and still unable to copy large files for same reason. anything else i can do?
UTM MacOS 13.5.1 Intel based Macbook Pro running Win7 x64

**Reply by mmgordon82** [reply]
Try restarting after making the changes. Worked for me.

**Reply by pivendren** [reply]
Thanks! Took a restart for me 🙏

**Comment by m4r1k** [comment]
Pretty wild this is not default as part of the UTM drivers ..

**Reply by slalomsquid** [reply]
yeah i know, seems a very common issue

**Comment by eaftab** [comment]
it worked for me but now suddenly stopped working, no matter how many times i change the value i am not able to get it working.

**Comment by tinoargentino** [comment]
The `FileSizeLimitinBytes` failed for me, fixed it by first copying the iso to the thumb drive from my mac, ejecting, then within utm connecting the thumb, drag dropping the iso into the desktop, and then loading it into rufus from there

**Comment by ahmaabdo** [comment]
Worked for me too, here's a CMD as admin command
`reg add "HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\WebClient\Parameters" /v FileSizeLimitInBytes /t REG_DWORD /d 4294967295 /f`
